tests/helpers/__init__.py

```diff
@@ -83,7 +82,7 @@ def get_marked_line_numbers(path):
     with open(path) as f:
         lines = {}
         for i, line in enumerate(f):
-            match = re.search(r'#@\s*(.*?)\s*$', line)
+            match = re.search(r'#\s*@\s*(.*?)\s*$', line)
```

Comment:
The requirement to use `#@` was intentional, to avoid accidentally clashing with a regular comment. Is it causing some linting issues? Our Flake8 config is supposed to allow this.

Reply:
It's not a linting issue, it's just that I use the pydev code formatter (https://github.com/fabioz/PyDev.Formatter) for all my code to avoid having to manually format anything and it auto-formatted `#@` to `# @` (so, in general I don't worry about any linting issues and it's not a picky formatter as it tries to keep as close to the original code as possible).

Regarding clashing with a regular comment... even if it does get some extra lines, this doesn't seem like it'd be a problem in practice -- all the usages that I saw seem to access the keys by dict later on (so, if it's ok I'd like to do that change so that I can still use the formatter).